front/src/modules/ui/data/view-bar/components/MultipleFiltersDropdownButton.tsx

```diff
@@ -14,13 +17,15 @@ type MultipleFiltersDropdownButtonProps = {
 export const MultipleFiltersDropdownButton = ({
   hotkeyScope,
 }: MultipleFiltersDropdownButtonProps) => {
+  const dropdownWidth = useRecoilValue(dropdownWidthState);
```

Comment:
I think we would prefer the other way around. We do not get dropdownWidth and pass it as a props to Dropdown here.
Instead:
- we don't pass it (or we pass an initialization value ; as we already have a default in Dropdown, we don't actually need to pass anything).
- from the inside of Dropdown, we listen to dropdownWidthState for changes.

We will need to introduce a DropdownOnWidthChangeEffect very similar to DropdownOnToggleEffect